Dockerfiles/Dockerfile.bcdaworker_prod

```diff
@@ -1,4 +1,3 @@
-# syntax = docker/dockerfile:1-experimental
```

Comment:
Removed references to a different docker frontend, which slightly speeds up the image build. My theory is that this was originally added to enable run cache mounting.